sentinel-core/src/test/java/com/alibaba/csp/sentinel/eagleeye/EagleEyeCoreUtilsTest.java

```diff
@@ -0,0 +1,164 @@
+        EagleEyeCoreUtils.checkNotNullEmpty(" ", "bar");
+        EagleEyeCoreUtils.checkNotNullEmpty(null, "bar");
+        // Methods are not expected to return due to exception thrown
+    }
+
+    @Test
+    public void testCheckNotNull() {
+        Assert.assertEquals("", EagleEyeCoreUtils.checkNotNull("", "bar"));
+        Assert.assertEquals(" ", EagleEyeCoreUtils.checkNotNull(" ", "bar"));
+        Assert.assertEquals("foo",
+                EagleEyeCoreUtils.checkNotNull("foo", "bar"));
+
+        thrown.expect(IllegalArgumentException.class);
+        EagleEyeCoreUtils.checkNotNull(null, "bar");
+        // Method is not expected to return due to exception thrown
+    }
+
+    @Test
+    public void testDefaultIfNull() {
+        Assert.assertEquals("", EagleEyeCoreUtils.defaultIfNull("", "bar"));
+        Assert.assertEquals(" ", EagleEyeCoreUtils.defaultIfNull(" ", "bar"));
+        Assert.assertEquals("bar",
+                EagleEyeCoreUtils.defaultIfNull(null, "bar"));
+        Assert.assertEquals("foo",
+                EagleEyeCoreUtils.defaultIfNull("foo", "bar"));
+    }
+
+    @Test
+    public void testIsNotBlank() {
+        Assert.assertFalse(EagleEyeCoreUtils.isNotBlank(""));
+        Assert.assertFalse(EagleEyeCoreUtils.isNotBlank(" "));
+        Assert.assertFalse(EagleEyeCoreUtils.isNotBlank(null));
+
+        Assert.assertTrue(EagleEyeCoreUtils.isNotBlank("foo"));
+    }
+
+    @Test
+    public void testIsNotEmpty() {
+        Assert.assertFalse(EagleEyeCoreUtils.isNotEmpty(""));
+        Assert.assertFalse(EagleEyeCoreUtils.isNotEmpty(null));
+
+        Assert.assertTrue(EagleEyeCoreUtils.isNotEmpty(" "));
+        Assert.assertTrue(EagleEyeCoreUtils.isNotEmpty("foo"));
+    }
+
+    @Test
+    public void testTrim() {
+        Assert.assertNull(EagleEyeCoreUtils.trim(null));
+
+        Assert.assertEquals("", EagleEyeCoreUtils.trim(""));
+        Assert.assertEquals("", EagleEyeCoreUtils.trim(" "));
+        Assert.assertEquals("foo", EagleEyeCoreUtils.trim("foo"));
+        Assert.assertEquals("foo", EagleEyeCoreUtils.trim("foo "));
+        Assert.assertEquals("foo", EagleEyeCoreUtils.trim(" foo"));
+        Assert.assertEquals("foo", EagleEyeCoreUtils.trim(" foo "));
+        Assert.assertEquals("f o o", EagleEyeCoreUtils.trim(" f o o "));
+    }
+
+    @Test
+    public void testSplit() {
+        Assert.assertNull(EagleEyeCoreUtils.split(null, 'a'));
+
+        Assert.assertArrayEquals(new String[0],
+                EagleEyeCoreUtils.split("", ','));
+        Assert.assertArrayEquals(new String[]{"foo", "bar", "baz"},
+                EagleEyeCoreUtils.split("foo,bar,baz", ','));
+    }
+
+    @Test
+    public void testAppendWithBlankCheck() {
+        Assert.assertEquals(new StringBuilder("bar").toString(),
+                EagleEyeCoreUtils.appendWithBlankCheck(
+                        null, "bar", new StringBuilder()).toString());
+        Assert.assertEquals(new StringBuilder("foo").toString(),
```

Comment:
Can the expected value not just be the string eg. `"bar"` instead of `new StringBuilder("bar").toString()`
(same with other similar tests)

Reply:
You are right.. I have no idea what I was doing here -_-